### ISSUE TITLE
Sneak bug with slabs and blocks.

### ISSUE THREAD

**Issue by IrrlichtMt4** [minetest/minetest#12702]
Build this
![screenshot_20220815_102606](https://user-images.githubusercontent.com/102578125/184560789-ada54ee5-d789-43fc-b877-41279b7bc20b.png)
![screenshot_20220815_102615](https://user-images.githubusercontent.com/102578125/184560792-f493aab8-d584-4c48-9e2c-97c133f618f3.png)

Go inside the hole and try to sneak on the slab while inside the hole. Players are still able o sneak on the slabs.

**Comment by Zughy** [comment]
I'm pretty sure is the same of #11532

**Comment by Zughy** [comment]
Yes, it's the same, but slabs allow you to go higher than two blocks

**Comment by v-rob** [comment]
This appears to be the main issue for sneak glitch stuff, so here's some more ~~fun~~ buggy behavior. You can sneak up to the top of this structure, despite it being two blocks high at the top.
![image](https://github.com/user-attachments/assets/b0e81690-8eea-4317-89e9-1572799a6da7)

**Comment by tigercoding56** [comment]
mark this as a feature and release it as sneak bug 2.0

**Comment by birdlover32767** [comment]
with the sneak glitch you can also float above blocks by slowly sneaking down a block (walk to its edge, quickly release and hold sneak, walk away from the block to see if you can walk)

this way you can jump across gaps you normally can't, like this one:

<img width="1920" height="1008" alt="Image" src="https://github.com/user-attachments/assets/f809a039-3946-4e43-b9fc-3d717e78a7fc" />

**Comment by SmallJoker** [comment]
@birdlover32767 That is a sneak glitch feature. Relevant code:
https://github.com/luanti-org/luanti/blob/36b5374715f1aae7fb1ca6fabe4ea4a24895512e/src/client/localplayer.cpp#L409-L416

----

The original issue reported here is about the following code:
https://github.com/luanti-org/luanti/blob/36b5374715f1aae7fb1ca6fabe4ea4a24895512e/src/client/localplayer.cpp#L92-L95
This value cannot be changed, but perhaps the logic improved to handle this special case.
EDIT: For those who don't know - this is a sneak ladder. I think it depends on the sneak glitch being enabled:

<img width="269" height="411" alt="Image" src="https://github.com/user-attachments/assets/e037ec3c-10ae-4686-a37b-8a96a8d983a6" />
